jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -798,12 +798,12 @@ properties:
             fits_keyword: GROUPGAP
             blend_table: True
           drop_frames1:
-            title: Frames dropped at start of each integration
+            title: Number of frames dropped prior to first integration
```

Comment:
The new value is a few too many characters long to fit within the comment field in the header without getting truncated (I believe the limit is 47 chars available for comment out of the total 80 char record; the others are occupied by the keyword name and its value). So let's shorten this a little bit to "Frames dropped prior to first integration". That's close to the KWD version.